mlir/include/mlir/Dialect/DLTI/DLTI.h

```diff
@@ -28,6 +28,12 @@ namespace dlti {
 /// query interface-implementing attrs, starting from attr obtained from `op`.
 FailureOr<Attribute> query(Operation *op, ArrayRef<DataLayoutEntryKey> keys,
                            bool emitError = false);
+
+/// Perform a DLTI-query at `op` using string `keys` as DLTI entry keys,
```

Comment:
nit: "string `keys`" seems to imply `keys` is a single string

Reply:
Updated description, should be clearer now